src/components/cc-toggle/cc-toggle.js

```diff
@@ -51,6 +51,7 @@ export class CcToggle extends LitElement {
       inline: { type: Boolean, reflect: true },
       legend: { type: String },
       multipleValues: { type: Array, attribute: 'multiple-values', reflect: true },
+      name: { type: String, reflect: true },
```

Comment:
issue(mild): The reflect is needed when we use the property in CSS selector. In this situation, we can remove it.

Reply:
I would prefer we keep reflecting it.

The name attribute is useful in a native context where element internals is not supported to retrieve elements participating in a form:

```js
const myFormElements = form.querySelectorAll([name])
```

We also use the name attribute in our experimentations with @pdesoyres-cc :wink: